src/config/featured.json

```
@@ -14,6 +14,13 @@
       "description": "KNIX MicroFunctions is a serverless computing platform that combines container-based resource isolation with a lightweight execution model using processes to significantly improve resource efficiency and decrease the function startup latency. KNIX MicroFunctions works in Knative as well as bare metal or virtual machine-based environments.",
       "logo": "https://raw.githubusercontent.com/knix-microfunctions/knix/develop/logo/KNIX_logo_stack.jpg"
     },
+    {
+      "name": "NTT",
+      "repo_url": "https://github.com/nokia/ntt",
+      "homepage": null,
+      "description": "Modern tools for TTCN-3. ntt gives you the tools you need to write tests efficiently.",
+      "logo": null
```

The least ugly we have ;)
```suggestion
      "logo": "https://nokia.github.io/ntt/static/ntt-mono-blue.png"
```